lib/binarisYML.js

```diff
@@ -51,7 +51,7 @@ const getFuncName = function getFuncName(binarisConf) {
 const checkFuncConf = async function checkFuncConf(funcConf, funcDirPath) {
   for (const section of [fileStr, entryStr, runtimeStr]) {
     if (!funcConf[section]) {
-      throw new Error(`${binarisConfFile}: missing field <${section}>`);
+      throw new Error(`${binarisConfFile}: missing a mandatory field <${section}>`);
```

Comment:
The Google Search Academy of the English Language (URLs https://google.com/search?q=%22required+field%22 and https://google.com/search?q=%22mandatory+field%22) suggest that this field is "required".